packages/astro/components/ViewTransitions.astro

```diff
@@ -276,6 +276,7 @@ const { fallback = 'animate' } = Astro.props as Props;
 			if (
 				link &&
 				link instanceof HTMLAnchorElement &&
+				(link.dataset.astroReload ?? 'false') === 'false' &&
```

Comment:
is there a reason why someone would use a false value here? If not I think we can just check for it's existence to save a few bytes.

Reply:
For me it looks odd if someone writes: `reload="false"` and gets a reload. 
But at the end it is a matter of documentation: "Any value assigned to this attribute is ignored."
10 Bytes saved :-) !